webpack/index.d.ts

```diff
@@ -1020,7 +1020,18 @@ declare namespace webpack {
             new (preferEntry: boolean): Plugin;
         }
         interface UglifyJsPluginStatic {
-            new (options?: UglifyJS.MinifyOptions): Plugin;
+            new (
+              options?: UglifyJS.MinifyOptions,
+              compress?: boolean | UglifyJS.CompressorOptions,
```

Comment:
Aren't `compress`, `mangle`, etc.  keys in `UglifyJS.MinifyOptions`? Looking at the [docs](https://webpack.github.io/docs/list-of-plugins.html#uglifyjsplugin),  I don't think these are passed in as params.